src/BenchmarkDotNet/Portability/RuntimeInformation.cs

```diff
@@ -42,6 +42,13 @@ public static bool IsCoreRT
             => ((Environment.Version.Major >= 5) || FrameworkDescription.StartsWith(".NET Core", StringComparison.OrdinalIgnoreCase))
                && string.IsNullOrEmpty(typeof(object).Assembly.Location); // but it's merged to a single .exe and .Location returns null here ;)
 
+        /// <summary>
+        /// "Is the target where we will run the benchmarks WASM?"
+        /// </summary>
+        public static bool IsBenchmarkTargetWasm { get; set; }
+
+        public static bool IsWasm() => IsOSPlatform(OSPlatform.Create("BROWSER")); // { return BenchmarkEnvironmentInfo.RunningWasm; }
```

Comment:
nit: outdated comment

```suggestion
        public static bool IsWasm() => IsOSPlatform(OSPlatform.Create("BROWSER"));
```